### PR TITLE
add some utils

### PR DESCRIPTION
**Pull request by Yutyo**

### CODE REVIEW

File: config/base/software.nix [hunk 1]
```diff
@@ -33,12 +33,14 @@ in
...
     gnome3.gnome-disk-utility
+    ark
+    psmisc
+    htop
+    tree
 
     # apps
     cinnamon.nemo
     libreoffice
-    flameshot
```
Comment (mkg20001) [inline config/base/software.nix:40]: that one is actually useful

File: config/base/software.nix [hunk 2]
```diff
@@ -33,12 +33,14 @@ in
     curl
     wget
     gnome3.gnome-disk-utility
+    ark
```
Comment (mkg20001) [inline config/base/software.nix:36]: also we should add why we added each tool, so we can think about whether the choice still makes sense

File: config/base/software.nix [hunk 3]
```diff
@@ -49,9 +51,6 @@ in
     merculator
     distrocards
 
-    # utils
-    htop
-
     # browsers
```
Comment (mkg20001) [inline config/base/software.nix:54]: like ẃe had categries some time ago

File: config/base/software.nix [hunk 4]
```diff
@@ -33,12 +33,14 @@ in
     curl
     wget
     gnome3.gnome-disk-utility
+    ark
+    psmisc
+    htop
+    tree
```
Comment (mkg20001) [inline config/base/software.nix:40]: you wanted to add killall aswell

File: config/base/software.nix [hunk 5]
```diff
@@ -37,6 +37,8 @@ in
     psmisc
     htop
     tree
+    gnome3.nautilus
```
Comment (mkg20001) [inline config/base/software.nix:40]: we have nemo already. what's the point?

File: config/base/software.nix (hunk 2 above)
Comment (Yutyo) [reply config/base/software.nix:36]: Those are essential for an user. A thing such as Ark must be there. Just think about those Ubuntu flavours such as Xubuntu, Ubuntu Mate etc. that have those as well. Our priority is being user friendly.

File: config/base/software.nix (hunk 4 above)
Comment (Yutyo) [reply config/base/software.nix:42]: Killall is within psmisc

File: config/base/software.nix (hunk 5 above)
Comment (Yutyo) [reply config/base/software.nix:40]: Lol, yes.

File: config/base/software.nix (hunk 2 above)
Comment (mkg20001) [reply config/base/software.nix:36]: We should add a comment for every tool WHY we added it

For example
`flameshot # screenshot tool that supports editing the screenshot in-place`

Comment (mkg20001) [reply config/base/software.nix:36]: Also there is file-roller aswell, ark is kde based while file-roller is gtk based

You need to ask yourself more questions...

File: config/base/software.nix (hunk 4 above)
Comment (mkg20001) [reply config/base/software.nix:42]: K, nice

**Comment by Yutyo** [comment]
I have pushed skel changes there as well.

File: pkgs/meros-skel-xfce/skel/.config/xfce4/terminal/terminalrc [hunk 6]
```diff
@@ -26,6 +26,11 @@ MiscSlimTabs=FALSE
...
 MiscSearchDialogOpacity=100
 MiscShowUnsafePasteDialog=TRUE
+BackgroundDarkness=0,850000
+ColorPalette=rgb(0,0,0);rgb(170,0,0);rgb(255,77,13);rgb(170,85,0);rgb(0,0,170);rgb(170,0,170);rgb(0,170,170);rgb(170,170,170);rgb(85,85,85);rgb(255,85,85);rgb(85,255,85);rgb(255,255,85);rgb(85,85,255);rgb(255,85,255);rgb(85,255,255);rgb(255,255,255)
+ColorBoldIsBright=FALSE
+TabActivityColor=#ffff4d4d0d0d
+ColorBackground=#000000000000
 BackgroundMode=TERMINAL_BACKGROUND_TRANSPARENT
-BackgroundDarkness=0.750000
+BackgroundImageFile=/home/yutyo/Pictures/billgatesangry.jpg
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/.config/xfce4/terminal/terminalrc:35]: :thinking:

File: config/base/software.nix (hunk 1 above)
Comment (Yutyo) [reply config/base/software.nix:40]: Misstyped.

**Comment by Yutyo** [comment]
Its OK now, I guess?

File: config/base/software.nix [hunk 7]
```diff
@@ -21,43 +21,44 @@ in
...
-    flameshot
-    atom
-    wine
-    shotwell
-    font-manager
-    qpaeq
-    cinnamon.warp
+    cinnamon.nemo                # To show files
+    libreoffice                  # Office program that comes with literally every GNU/Linux distro
+    wine                         # Tool for people that wanna run Windows programs instead of glorious free software
```
Comment (mkg20001) [inline config/base/software.nix:45]: I don't know if we should make this optional since wine is huge (1gb)

File: config/base/software.nix [hunk 8]
```diff
@@ -21,43 +21,44 @@ in
...
+    distrocards                  # Magnificent card game of the free people of Mercode
 
     # browsers
     # chromium
-    firefox
+    firefox                      # The browser that comes bundled with every GNU/Linux distro except for ChromeOS and Raspbian
 
     # autostart
-    (mkAutostart ["flameshot"])
+    (mkAutostart ["flameshot"])  # You know, Flameshot autostarts at MerOS
```
Comment (mkg20001) [inline config/base/software.nix:61]:
```suggestion
    (mkAutostart ["flameshot"])  # autostart flameshot with the OS
```

File: pkgs/meros-skel-xfce/skel/panel/battery-11.rc [hunk 9]
```diff
@@ -0,0 +1,20 @@
+display_label=false
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/panel/battery-11.rc:1]: this relates to a sepcific battery, thus you shouldn't have it here

File: pkgs/meros-skel-xfce/skel/panel/battery-12.rc [hunk 10]
```diff
@@ -0,0 +1,20 @@
+display_label=false
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/panel/battery-12.rc:1]: as above

File: config/base/software.nix [hunk 11]
```diff
@@ -21,43 +21,44 @@ in
...
     # external package formats
-    flatpak
+    flatpak                      # Good software utility to use if you can't find what you want at nixpkgs
     # snapd N/A yet
 
     # sys apps
-    conf-tool
-    nixNodePackage
+    conf-tool                    # Config Tool
+    nixNodePackage               # NodeJS pre-installed
```
Comment (mkg20001) [inline config/base/software.nix:29]: not really. actually:

```suggestion
    nixNodePackage               # hack to get nix-node-package to be available without downloading it at every eval
```

File: config/base/software.nix [hunk 12]
```diff
@@ -21,43 +21,44 @@ in
...
 
     # utilities
-    git
-    curl
-    wget
-    gnome3.gnome-disk-utility
+    git                          # That's git everybody, our favourite scm
+    curl                         # Ancient way of transfering data
```
Comment (mkg20001) [inline config/base/software.nix:33]: wget is the more ancient one, actually

Comment (mkg20001) [reply config/base/software.nix:33]: curl is mostly for devs, wget is mostly for people who want to download stuff
```suggestion
    curl                         # HTTP-debugger
```

File: config/base/software.nix [hunk 13]
```diff
@@ -21,43 +21,44 @@ in
...
+    wget                         # Less ancient way of transfering basic web data
```
Comment (mkg20001) [inline config/base/software.nix:34]:
```suggestion
    wget                         # The good ol' downloader
```

File: pkgs/meros-skel-xfce/skel/panel/launcher-10/15810088764.desktop [hunk 14]
```diff
@@ -2,7 +2,7 @@
 Name=Atom
 Comment=A hackable text editor for the 21st Century.
 GenericName=Text Editor
-Exec=/nix/store/c9pr6v3qhksdgvx9h3j2c0mkf10k2hza-atom-1.42.0/bin/atom
+Exec=/nix/store/vrrag98zig6ymp32gik0sb7v1lavz21f-atom-1.42.0/bin/atom
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/panel/launcher-10/15810088764.desktop:5]: this breaks, make it a /run/current-system/sw/bin/ path

File: pkgs/meros-skel-xfce/skel/panel/launcher-11/15810088865.desktop [hunk 15]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Exec=Discord
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/panel/launcher-11/15810088865.desktop:3]: discord? really?

File: pkgs/meros-skel-xfce/skel/panel/launcher-12/15810088946.desktop [hunk 16]
```diff
@@ -0,0 +1,15 @@
+[Desktop Entry]
+Version=1.0
+Name=Telegram Desktop
+Comment=Official desktop version of Telegram messaging app
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/panel/launcher-12/15810088946.desktop:4]: also we shouldn't add those. only basic apps, telegram isn't preinstalled anyways

File: pkgs/meros-skel-xfce/skel/panel/launcher-7/15810088331.desktop [hunk 17]
```diff
@@ -0,0 +1,16 @@
...
+Terminal=false
+Categories=Utility;X-XFCE;X-Xfce-Toplevel;
+OnlyShowIn=XFCE;
+X-XFCE-MimeType=inode/directory;x-scheme-handler/trash;
+X-AppStream-Ignore=True
+Name=File Manager
+Name[tr]=Dosya Yöneticisi
+Comment=Browse the file system
+Comment[tr]=Dosya sistemini tara.
+X-XFCE-Source=file:///nix/store/swbv7k9qj96h523dysvvmdzkzx3zgsiw-exo-0.12.8/share/applications/exo-file-manager.desktop
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/panel/launcher-7/15810088331.desktop:16]: this breaks, make it a /run/current-system/sw/bin/ path

File: pkgs/meros-skel-xfce/skel/panel/launcher-7/15810909981.desktop [hunk 18]
```diff
@@ -0,0 +1,16 @@
...
+Terminal=false
+Categories=Utility;X-XFCE;X-Xfce-Toplevel;
+OnlyShowIn=XFCE;
+X-XFCE-MimeType=inode/directory;x-scheme-handler/trash;
+X-AppStream-Ignore=True
+Name=File Manager
+Name[tr]=Dosya Yöneticisi
+Comment=Browse the file system
+Comment[tr]=Dosya sistemini tara.
+X-XFCE-Source=file:///nix/store/swbv7k9qj96h523dysvvmdzkzx3zgsiw-exo-0.12.8/share/applications/exo-file-manager.desktop
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/panel/launcher-7/15810909981.desktop:16]: It seems like this is the third time this app shortcut is here

File: pkgs/meros-skel-xfce/skel/panel/launcher-8/15810930522.desktop [hunk 19]
```diff
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Exec=Discord
+Terminal=false
+Name=Discord
+Categories=Network;InstantMessaging;
+Icon=discord
+GenericName=All-in-one cross-platform voice and text chat for gamers
+MimeType=x-scheme-handler/discord
+X-XFCE-Source=file:///home/yutyo/.nix-profile/share/applications/discord.desktop
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/panel/launcher-8/15810930522.desktop:10]: also no

File: pkgs/meros-skel-xfce/skel/panel/whiskermenu-1.rc [hunk 20]
```diff
@@ -0,0 +1,80 @@
+favorites=Thunar.desktop
+recent=gimp.desktop,telegramdesktop.desktop,discord.desktop,pavucontrol.desktop,skypeforlinux.desktop,com.uploadedlobster.peek.desktop,firefox.desktop,Thunar.desktop,emacs.desktop,distrocards.desktop
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/panel/whiskermenu-1.rc:2]: remove that line, or reduce this to the minimum

File: pkgs/meros-skel-xfce/skel/xfconf/xfce-perchannel-xml/parole.xml [hunk 21]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<channel name="parole" version="1.0">
+  <property name="subtitles" type="empty">
+    <property name="font" type="string" value="Sans Bold 20"/>
+  </property>
+  <property name="folders" type="empty">
+    <property name="last-used-media" type="string" value="/home/yutyo/Downloads/flappy/empty/assets"/>
```
Comment (mkg20001) [inline pkgs/meros-skel-xfce/skel/xfconf/xfce-perchannel-xml/parole.xml:8]: :thinking:

File: config/base/software.nix (hunk 7 above)
Comment (Yutyo) [reply config/base/software.nix:45]: Yes, wine shouldn't come pre-installed. Its for people that want proprietary .exe programs to work, yet we wanna encourage people to use free software.

Comment (mkg20001) [reply config/base/software.nix:45]: You could make a bundle, look in `config/base/bundles.nix`

File: config/base/software.nix [hunk 22]
```diff
@@ -21,43 +21,44 @@ in
...
+    gnome3.gnome-disk-utility    # As the name says
+    file-roller                  # A WinRAR that doesn't ask you to pay
+    psmisc                       # Needed utils bundled in a single package
+    htop                         # Interactive process viewer
+    tree                         # A better version of ls
```
Comment (mkg20001) [inline config/base/software.nix:39]: we also need lesspipe for scrolled output
it's required to be in bin now, since the bashrc will check for it

**Comment by Yutyo** [comment]
OK, now that is just a PR for utils.

**Pull request merged**